### DISCUSSION TITLE
Run renovate to trigger updates for different groups of dependencies at different times

### DISCUSSION THREAD

**Discussion by priyankajain1711** [renovatebot/renovate#22727] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

GitHub Enterprise Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

We have 4 groups of dependencies which I want to update on demand on different days using Jenkins. 
Currently we created 4 different config.js. However when we run with 1 config.js  the PRs from other config are deleted. 

Can this be achieved using Renovate

### What you tried so far.

the groups created work completely well when triggered together, but requirement is to update this dependency on demand for a group

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
You can define different `schedule` values for difference dependencies using packageRules

**Reply by priyankajain1711** [reply]
But the schedule is not definite. it can be on demand also. Is there a way to achieve it

**Reply by rarkins** [reply]
I don't understand how that could work

**Reply by priyankajain1711** [reply]
Is there a way to stop renovate from closing the Old PRs from a different config

**Reply by rarkins** [reply, marked as answer]
https://docs.renovatebot.com/configuration-options/#prunestalebranches

**Reply by priyankajain1711** [reply]
this closes the older PRs from a different config, Have tried it

**Reply by rarkins** [reply]
That should not be the case

**Reply by priyankajain1711** [reply]
pruneStableBranches works , we enabled that and running with different config.js files for different groups

**Comment by priyankajain1711** [comment]
Is there any other solution to above problem with renovate where we can run only for one group of dependencies
